### ISSUE TITLE
[Kubernetes Operator] Integrate `poktrolld` with k8s Operator and `heighliner`

### ISSUE THREAD

**Issue by okdas** [pokt-network/poktroll#578]
## Objective

Integrate new version of Pocket Network's validator - `poktrolld` with the Cosmos-SDK Kubernetes Operator.

## Origin Document

This integration will ensure seamless deployment and management of the new `poktrolld` Shannon upgrade within Kubernetes environments using the Cosmos-SDK operator. [Cosmos Operator GitHub Repository](https://github.com/strangelove-ventures/cosmos-operator)

#581 should unblock most of this work.

### Secondary details from #581 (duplicate):
- Strangelove implemented `heighliner`: 
	> We built Heighliner to streamline the management and building of production-grade container images for blockchain node software, Cosmos and beyond.
- It is a precusor for [strangelove-ventures/cosmos-operator](https://github.com/strangelove-ventures/cosmos-operator)
- @okdas attempted the integration in #519
- @okdas put the integration on pause per the comment [here](https://github.com/pokt-network/poktroll/pull/523#issuecomment-2103668830) in #523

## Goals

- Ensure compatibility between `poktrolld` Shannon upgrade and Cosmos-SDK Kubernetes Operator.
- Simplify the deployment process of `poktrolld` using the operator.
- Integrate Shannon with tools in the broader cosmos ecosystem
- Make it easier for others to deploy POKT actors

## Deliverables

- [ ] Validate the compatibility of `poktrolld` Shannon upgrade with the Cosmos-SDK Kubernetes Operator.
- [ ] Document the deployment process for `poktrolld` using the Cosmos-SDK Kubernetes Operator.
- [ ] Test the integration in a staging environment (can be validated/performed by Grove engineering team).

## Non-goals / Non-deliverables

- Modify the core functionalities of the Cosmos-SDK Kubernetes Operator.
- Provide support for non-Kubernetes deployment environments.

## General deliverables

- [ ] **Comments**: Add/update TODOs and comments alongside the source code so it is easier to follow.
- [ ] **Testing**: Add new tests (unit and/or E2E) to the test suite.
- [ ] **Makefile**: Add new targets to the Makefile to make the new functionality easier to use.
- [ ] **Documentation**: Update architectural or development READMEs; use [mermaid](https://mermaid-js.github.io/mermaid/) diagrams where appropriate.

---

**Creator**: @okdas
**Co-Owners**: TBD

**Comment by Olshansk** [comment]
@okdas I believe you tried this but am not sure if we still plan on doing it given the state w/ cosmosvisor et al.

If we're not going to do it, please close it out.

**Comment by okdas** [comment]
We've got feedback that people are actually interested in this - will look into cosmos operator now.

**Comment by Olshansk** [comment]
Closing this out. If it becomes a priority, we'll do it.
